libdevice/cmath_wrapper.cpp

```diff
@@ -23,12 +23,10 @@ DEVICE_EXTERN_C
 div_t div(int x, int y) { return __devicelib_div(x, y); }
 
 DEVICE_EXTERN_C
-ldiv_t ldiv(long int x, long int y) { return __devicelib_ldiv(x, y); }
+ldiv_t ldiv(long x, long y) { return __devicelib_ldiv(x, y); }
 
 DEVICE_EXTERN_C
-lldiv_t lldiv(long long int x, long long int y) {
-  return __devicelib_lldiv(x, y);
-}
+lldiv_t lldiv(long long x, long long y) { return __devicelib_lldiv(x, y); }
 
```

Comment:
What's the purpose of these changes?

Reply:
Just to align with math.h prototype, a minor NFC change.

Thanks very much.